Projects/Server/main.cpp

```diff
@@ -8,19 +9,18 @@
 
 int main()
 {
-    GameHandler gameHandler{};
-
     //AbstractPlayer::Ptr player0 = std::make_shared<StackAI::StackPlayer>();
-    //AbstractPlayer::Ptr player1 = std::make_shared<TranqAI::TranqPlayer>();
+    AbstractPlayer::Ptr player1 = std::make_shared<TranqAI::TranqPlayer>();
 
     //AbstractPlayer::Ptr player0 = std::make_shared<RandomPlayer>();
     AbstractPlayer::Ptr player0 = std::make_shared<InteractivePlayer>();
-    AbstractPlayer::Ptr player1 = std::make_shared<RandomPlayer>();
-
-    gameHandler.init(player0, player1);
-
-    //for (size_t repeat = 0; repeat < 10; ++repeat)
-        gameHandler.play();
-
+    //AbstractPlayer::Ptr player1 = std::make_shared<RandomPlayer>();
+
+    GameCounter gameCounter{ player0, player1, 1 };
```

Comment:
`GameCounter` is misleading, something like `MultiGameRunner` is more on point. (Actually anything that points at 'this will not run the games, but will arrange multiple games' should work.)

Reply:
Changed the class name to your suggestion `MultiGameRunner`